### PR TITLE
feat(batch-requests): allow customizing uri in batch-requests

### PR DESCRIPTION
**Pull request by arthur-zhang**
### What this PR does / why we need it:
Support  customizing uri in batch-requests like this

```
plugin_attr:
  batch-requests:
    uri: "/api-gw/batch"
```
we can send batch request like this

```
curl --location --request POST 'http://localhost:9080/api-gw/batch' \
--header 'Content-Type: application/json' \
--data-raw '{
    "headers": {
        "Content-Type": "application/json"
    },
    "timeout": 5000,
    "pipeline": [
        {
            "method": "GET",
            "path": "/go"
        }
    ]
}'
```

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by tokers** [comment]
@arthur-zhang Need to check out the CI problems.

**Comment by arthur-zhang** [comment]
> @arthur-zhang Need to check out the CI problems.

I am working on it.

**Comment by arthur-zhang** [comment]
> @arthur-zhang Need to check out the CI problems.

all done

File: t/plugin/batch-requests.t [hunk 1]
```diff
@@ -1017,3 +1017,204 @@ GET /t
...
+
+
+
+=== TEST 24: customize uri, not found
+--- yaml_config
+plugin_attr:
+    batch-requests:
+        uri: "/foo/bar"
+--- config
+    location = /aggregate {
```
Comment (Yiyiyimu) [inline t/plugin/batch-requests.t:1029]: We used to name it `/t`

File: t/plugin/batch-requests.t [hunk 2]
```diff
@@ -1017,3 +1017,204 @@ GET /t
...
+--- no_error_log
+[error]
+
+
+
+=== TEST 26: customize uri, missing plugin, use default
+--- yaml_config
+plugin_attr:
+    x:
+      xyz: "/foo/bar"
```
Comment (Yiyiyimu) [inline t/plugin/batch-requests.t:1140]:
```suggestion
      uri: "/foo/bar"
```

File: docs/en/latest/plugins/batch-requests.md [hunk 3]
```diff
@@ -109,6 +109,22 @@ Response is `Array` of [HttpResponse](#httpresponse).
 | body    | string  | http response body    |
 | headers | object  | http response headers |
 
+## How to specify custom uri
+
+We can change the default uri in the `plugin_attr` section of `conf/config.yaml`.
+
+| Name       | Type   | Default                      | Description                       |
```
Comment (Yiyiyimu) [inline docs/en/latest/plugins/batch-requests.md:116]: Could you also add column `Requirement` like http://apisix.apache.org/docs/apisix/plugins/proxy-rewrite/#attributes

Comment (arthur-zhang) [reply docs/en/latest/plugins/batch-requests.md:116]: good suggestion

**Review by Yiyiyimu — APPROVED** [review]

File: docs/zh/latest/plugins/batch-requests.md [hunk 4]
```diff
@@ -114,6 +114,22 @@ curl http://127.0.0.1:9080/apisix/admin/plugin_metadata/batch-requests -H 'X-API
 | body    | string  | Http 请求的响应体   |
 | headers | object  | Http 请求的响应头   |
 
+## 如何修改自定义 uri
+
+我们可以在 `conf/config.yaml` 的 `plugin_attr` 修改默认的 `uri`
```
Comment (tokers) [inline docs/zh/latest/plugins/batch-requests.md:119]:
```suggestion
我们可以在 `conf/config.yaml` 的 `plugin_attr` 配置项中修改默认的 `uri`
```

File: t/plugin/batch-requests.t [hunk 5]
```diff
@@ -1017,3 +1017,204 @@ GET /t
...
+    }
+--- request
+GET /t
+--- error_code: 404
+--- no_error_log
+[error]
+
+
+
+=== TEST 25: customize uri, found
```
Comment (spacewander) [inline t/plugin/batch-requests.t:1067]: The `t/plugin/batch-requests.t` is long enough. Let's add test in a new file `t/plugin/batch-requests2.t`.

File: t/plugin/batch-requests.t [hunk 6]
```diff
@@ -1017,3 +1017,204 @@ GET /t
...
+    }
+    location = /d {
+        content_by_lua_block {
+            ngx.status = 202
+        }
+    }
+--- request
+GET /aggregate
+--- error_code: 200
+--- no_error_log
```
Comment (spacewander) [inline t/plugin/batch-requests.t:1131]: We can avoid repeating it via https://github.com/apache/apisix/blob/42ab6149272920bbab66985b90e2754a7c6cb747/t/admin/upstream2.t#L32

File: t/plugin/batch-requests2.t [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+        }
+    }
+
+    location = /a {
+        content_by_lua_block {
+            ngx.status = 200
+        }
+    }
+--- request
+GET /t
```
Comment (spacewander) [inline t/plugin/batch-requests2.t:77]: The `GET /t` is already set.

File: t/plugin/batch-requests2.t [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+use t::APISIX 'no_plan';
+
+repeat_each(1);
+no_long_string();
+no_root_location();
+log_level("debug");
```
Comment (spacewander) [inline t/plugin/batch-requests2.t:22]: We don't need to set log level to debug

**Review by spacewander — APPROVED** [review]

**Pull request merged**